### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by harry1453**
This PR contains fixes for various issues raised during release preparation for `v2.3.0-beta1`.

### CODE REVIEW

File: conf/brs-default.properties [hunk 1]
```diff
@@ -127,6 +127,9 @@ brs.blockCacheMB = 40
 # Accept http/json API requests.
 API.Server = on
 
+# Accept gRPC API requests.
+API.V2.Server = on
```
Comment (blankey1337) [inline conf/brs-default.properties]: 💥

File: src/brs/db/sql/SqlAccountStore.java [hunk 2]
```diff
@@ -111,33 +111,16 @@ protected Account load(DSLContext ctx, ResultSet rs) throws SQLException {
 
       @Override
       protected void bulkInsert(DSLContext ctx, ArrayList<Account> accounts) {
-        if ( ctx.fetchExists(ctx.selectOne().from(ACCOUNT).where(ACCOUNT.HEIGHT.eq(Burst.getBlockchain().getHeight())).limit(1)) ) {
```
Comment (harry1453) [inline src/brs/db/sql/SqlAccountStore.java]: This line was hitting sync performance really hard. It's more efficient to not have this check at all.

File: src/brs/db/sql/SqlAccountStore.java [hunk 3]
```diff
@@ -111,33 +111,16 @@ protected Account load(DSLContext ctx, ResultSet rs) throws SQLException {
...
+        ArrayList<Query> accountQueries = new ArrayList<>();
+        for ( Account account: accounts ) {
+          accountQueries.add(
+                  ctx.mergeInto(ACCOUNT, ACCOUNT.ID, ACCOUNT.HEIGHT, ACCOUNT.CREATION_HEIGHT, ACCOUNT.PUBLIC_KEY, ACCOUNT.KEY_HEIGHT, ACCOUNT.BALANCE,
+                          ACCOUNT.UNCONFIRMED_BALANCE, ACCOUNT.FORGED_BALANCE, ACCOUNT.NAME, ACCOUNT.DESCRIPTION, ACCOUNT.LATEST)
+                          .key(ACCOUNT.ID, ACCOUNT.HEIGHT).values(account.getId(), Burst.getBlockchain().getHeight(), account.getCreationHeight(), account.getPublicKey(), account.getKeyHeight(),
+                          account.getBalanceNQT(), account.getUnconfirmedBalanceNQT(), account.getForgedBalanceNQT(), account.getName(), account.getDescription(), true)
+          );
         }
+        ctx.batch(accountQueries).execute();
```
Comment (blankey1337) [inline src/brs/db/sql/SqlAccountStore.java]: Very nice

**Comment by blankey1337** [comment]
LGTM

**Comment by harry1453** [comment]
Merging without 2 approvals as we are about to deploy testnet release anyway so issues can be flagged at that stage.

**Pull request merged**